src/components/elements/Inputs/FileInput/FileInput.tsx

```diff
@@ -117,30 +125,54 @@ const FileInput = (props: FileInputProps) => {
             multiple={props.allowMultiple}
             accept={accept}
           />
-          <div className="m-auto flex w-fit items-center justify-center gap-3">
-            <FileCardContent
-              title={labelText}
-              subtitle={`${t("Drag and drop or")} <span className="text-primary underline">${t(
-                "browse your device"
-              )}</span>`}
-              thumbnailClassName="fill-primary"
-              thumbnailContainerClassName="bg-primary-100"
-            />
-          </div>
+          <When condition={!variant.snapshotPanel}>
+            <div className="m-auto flex w-fit items-center justify-center gap-3">
+              <FileCardContent
+                title={labelText}
+                subtitle={`${t("Drag and drop or")} <span className="text-primary underline">${t(
+                  "browse your device"
+                )}</span>`}
```

Comment:
This translation is problematic because it's making assumptions about the order of a sentence that only holds true for EN. I think this works and would be better:
```
subtitle={t("Drag and drop or {browse}", { browse: `<span className="text-primary underline">${t("browse your device")}</span>`)}
```

Reply:
done